tv.kodi.Kodi.json

```
@@ -0,0 +1,193 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "finish-args": [
+        "--device=all",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
+        "--filesystem=/mnt",
+        "--filesystem=/media",
+        "--filesystem=/run/media",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.Avahi"
```

Kodi wants to talk to UDisks and UPower, probably we should allow that.
```suggestion
        "--system-talk-name=org.freedesktop.Avahi",
        "--system-talk-name=org.freedesktop.UDisks2",
        "--system-talk-name=org.freedesktop.UPower"
```